templates/templates.js

```diff
@@ -60,4 +60,35 @@
       ["README.md"]
     ]
   }
+}, {
+  "name": "Empty AssemblyScript Project",
+  "description": `# Empty AssemblyScript Project
```

Comment:
The contents of this field is shown when selecting the template on the main screen. It's a good opportunity to describe what AssemblyScript is all about, maybe add a link back to the project, etc.